src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/subjectAccessRequest/SarLicenceStatus.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.subjectAccessRequest
+
+import com.fasterxml.jackson.annotation.JsonValue
+
+enum class SarLicenceStatus(@JsonValue val description: String) {
+  IN_PROGRESS("The licence is in progress and has not yet been submitted for approval"),
```

Comment:
Based on response from the HAAR team, we don't need to expose these long descriptions. 
We can just say "Inprogress", "Approved", "Variation submitted" etc..